lib/uploadcare/client/multipart_upload_client.rb

```diff
@@ -36,8 +36,10 @@ def upload_start(object, store: false)
       # When every chunk is uploaded, ask Uploadcare server to finish the upload
       def upload_complete(uuid)
         body = HTTP::FormData::Multipart.new(
-          'UPLOADCARE_PUB_KEY': Uploadcare.config.public_key,
-          'uuid': uuid
+          {
+            'UPLOADCARE_PUB_KEY': Uploadcare.config.public_key,
+            'uuid': uuid
+          }
```

Comment:
Why do we need this change?

Reply:
Because of ruby 3.0 support
The class `HTTP::FormData::Multipart` can only accept a Hash argument if ruby 3.0 is used.